src/front-end/src/components/MenteeForm.js

```diff
@@ -0,0 +1,109 @@
+/* eslint-disable */
+import React, { Component } from 'react';
+import { Form, Button, Input, DatePicker, Tooltip, Select } from 'antd';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { InfoCircleOutlined, QuestionCircleOutlined } from '@ant-design/icons';
+import TimezonePicker from 'react-bootstrap-timezone-picker';
+import 'react-bootstrap-timezone-picker/dist/react-bootstrap-timezone-picker.min.css';
+
+const { RangePicker } = DatePicker;
+
+const { Option } = Select;
+
+class MenteeForm extends Component {
+  render() {
+    return (
+    <div className="MenteeForm"> 
+      <Form>
+        <Form.Item
+          label="Name"
+          name="nameInput"
+          // must have an input:
+          rules={[{ required: true, message: 'Please input something' }]}
+        >
+          <Input placeholder="please enter your name" />
+        </Form.Item>
+        <Form.Item
+          label="Email"
+          name="emailInput"
+          // must have an input:
+          rules={[{ required: true, message: 'Please input something' }]}
+        >
+          <Input placeholder="please enter your email" />
+        </Form.Item>

```

Comment:
Under the name property, can you add "type="email"" so that the format of the user input is for an email?

Reply:
done